core/src/avm2/globals/flash/profiler/Telemetry.as

```diff
@@ -1,4 +1,5 @@
 package flash.profiler {
+    [API("678")]
```

Comment:
I think this should be `682`, for Flash Player 11.4:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/profiler/Telemetry.html